src/Components/Filter/ProductFilter.js

```diff
@@ -1,93 +1,74 @@
-            </li>
-            <li 
-            className="sidebarItem">
-                <input type="checkbox"  name="Brand-btn"/>Fila
-            </li>
-            <li 
-            className="sidebarItem">
-                <input type="checkbox"  name="Brand-btn"/>Hummel
-            </li>
-            <li 
-            className="sidebarItem">
-                <input type="checkbox"  name="Brand-btn"/>Asics
-            </li>
-            
-            <h3 
-            className="filter-items-heading">Rating</h3>
-            <li 
-            className="sidebarItem">
-                <input type="radio" 
-                className="filter-item"  name="sort-btn"/>4 stars & above
-            </li>
-            <li 
-            className="sidebarItem">
-                <input type="radio"  name="sort-btn"/>3 stars & above
-            </li>
-            <li 
-            className="sidebarItem">
-                <input type="radio" 
-                className="filter-item"  name="sort-btn"/>2 stars & above
-            </li>
-            <li 
-            className="sidebarItem">
-                <input type="radio"  name="sort-btn"/>1 stars & above
-            </li>
+                <li
+                    className="sidebarItem">
+                    <input type="checkbox" name="categories-btn" onChange={() => dispatch({ type: "CATEGORY", payload: "Men" })} />Men
+                </li>
+                <li
+                    className="sidebarItem">
+                    <input type="checkbox" name="categories-btn" onChange={() => dispatch({ type: "CATEGORY", payload: "Woman" })} />Woman
+                </li>
 
-            <hr></hr>
-            <button className="clear-btn">Clear Filter</button>
-            
-        </ul>
-    </div>
+                <h3 className="filter-items-heading">Rating</h3>
+                <li
+                    className="sidebarItem">
+                    <input type="radio"
+                        className="filter-item" name="rating-btn" checked={rating === "4"} onChange={() => dispatch({ type: "RATING", payload: "4" })} />4 stars & above
+                </li>
+                <li
+                    className="sidebarItem">
+                    <input type="radio" className="filter-item" checked={rating === "3"} name="rating-btn" onChange={() => dispatch({ type: "RATING", payload: "3" })} />3 stars & above
+                </li>
+                <li
+                    className="sidebarItem">
+                    <input type="radio"
+                        className="filter-item" name="rating-btn" checked={rating === "2"} onChange={() => dispatch({ type: "RATING", payload: "2" })} />2 stars & above
+                </li>
+                <li
+                    className="sidebarItem">
+                    <input type="radio" name="rating-btn" checked={rating === "1"} onChange={() => dispatch({ type: "RATING", payload: "1" })} />1 stars & above
+                </li>
```

Comment:
just a suggestion - instead of hardcoding each rating input,  you can have array of rating and using map method it can be rendered. it will shorten the code .

Reply:
okey I will do